<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QWidget" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Downloads/address-book.png</normaloff>../../../../Downloads/address-book.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="nameText">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="surnameEdit">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nameEdit">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="phoneEdit">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="emailEdit">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="addressText">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="phoneText">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Phone:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="surnameText">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Surname:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="emailText">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>E-mail:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="addressEdit">
             <property name="font">
              <font>
               <family>Lato</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="avatarPhoto">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>256</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="avatarLabel">
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Avatar</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadPhotoButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Load...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>surnameEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>loadPhotoButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
